test_team_guardiansofpii.py

```diff
@@ -12,6 +12,22 @@ def test_anonymize_text(self):
 
     def test_email_address(self):
         """Test email_address functionality"""
+     test_string = "My email address is example@example.com"
+    actual = anonymize_text(test_string, ['EMAIL'])
```

Comment:
In addition to the indent comment please look carefully at this [documentation](https://microsoft.github.io/presidio/supported_entities/) and look for EMAIL - you'll find the correct format for the recognizer